environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/response/EnvironmentBaseResponse.java

```diff
@@ -32,6 +32,9 @@
     @ApiModelProperty(EnvironmentModelDescription.CREATE_FREEIPA)
     private Boolean createFreeIpa = Boolean.TRUE;
 
+    @ApiModelProperty(EnvironmentModelDescription.FREEIPA_INSTANCES_PER_GROUP)
+    private Integer freeIpaInstancesPerGroup = 1;
```

Comment:
I think we should create a FreeipaResponse Object here because probably we will see more and more property here